examples/ui/text.rs

```diff
@@ -42,6 +42,7 @@ fn setup(mut commands: Commands, asset_server: Res<AssetServer>) {
                 style: TextStyle {
                     font_size: 60.0,
                     color: Color::WHITE,
+                    ..Default::default()
```

Comment:
It looks like text now has a small amount of extra "left-padding" now (master on top, this pr on bottom)

![image](https://user-images.githubusercontent.com/2694663/98425474-15c78c80-204a-11eb-87ee-7a7b26b137ac.png)

Any clue why that is?

Reply:
Here is how it renders in firefox, which makes me think the first one is correct (red bounds are a div container):
![image](https://user-images.githubusercontent.com/2694663/98426557-2843c500-204e-11eb-8fe5-721343b8d854.png)

Reply:
It's the side bearing that has been subtracted from `min_x` making the total text rectangle larger as well as increasing the distance from the left, because (if ab_glyph uses the same methodology), the origin already "includes" the side bearing so this has doubled up:
![](http://mirror.informatimago.com/next/developer.apple.com/documentation/mac/Text/graphics/FM-01.jpg)
taken from: http://mirror.informatimago.com/next/developer.apple.com/documentation/mac/Text/Text-186.html

i.e. for any glyph:
```
// left edge
min_x = origin_x;
// right edge
max_x = origin_x + advance;
// top edge
min_y = origin_y + ascent; // because down is positive in ab_glyph, this becomes `origin_y - ascent`
// bottom edge
max_y = origin_y - descent; // down is positive in ab_glyph, so this would be `origin_y + descent`, however, descent is already provided as a negative value in ab_glyph, so it would be `origin_y + -descent`
```

---

Simply removing ` - scaled_font.h_side_bearing(glyph.id)` from `glyph_brush.rs` and `pipeline.rs` appears to fix it.

---

Before: 
![image](https://user-images.githubusercontent.com/38416468/98439463-f35a6f80-212c-11eb-8474-3a9afb3aa1d0.png)

After:
![image](https://user-images.githubusercontent.com/38416468/98439840-944a2a00-212f-11eb-91dd-38f6e37f8b32.png)

_48px font (bevy/ab_glyph) equivalent to 40px font / 30pt font (CSS/Word)
400% zoom
background image is bevy with white text
foreground image is Word with red text (confirmed behaves the same in Firefox)_